asserts/asserts.go

```diff
@@ -994,6 +997,34 @@ func checkNoAuthority(assertType *AssertionType, headers map[string]interface{})
 	return nil
 }
 
+func checkJSON(assertType *AssertionType, body []byte) (err error) {
+	defer func() {
+		if err != nil {
+			err = fmt.Errorf("assertion %s: %v", assertType.Name, err)
+		}
+	}()
+
+	if body == nil {
+		return fmt.Errorf(`body must contain JSON`)
+	}
+
+	var val interface{}
+	if err := json.Unmarshal(body, &val); err != nil {
+		return fmt.Errorf("invalid JSON in body: %v", err)
+	}
+
+	formatted, err := json.MarshalIndent(val, "", "  ")
```

Comment:
This will rely on whatever the order of the keys `json.MarshalIndent` decides on, no? We probably need a test to verify that the JSON keys can be in any order.

Reply:
Hmm, the json package always sorts lexicographically and `jq` can use `--sort-keys`, so probably fine.